cinder/templates/volumes/volumes.yaml

```diff
@@ -0,0 +1,21 @@
+#TODO For vmware we should use the hypervisor definition to define volume nodes to avoid repeating config - consider moving hypervisor, volume, share and network hardware definitions into global section.
+{{- $context := . -}}
+{{- range $i, $volume := .Values.global.hypervisors_vmware -}}
+{{ tuple $context $volume |include "volume_vmware" }}
+---
+{{ tuple $context $volume |include "volume_vmware_configmap" }}
+{{ if lt $i (sub (len $context.Values.global.hypervisors_vmware) 1) }}
```

Comment:
This is kind of annoying and makes thing hard to read. I assume it has to be done this way for now but it would be better if every spec just carries a yaml above it.